pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/measurements/tests/Test_MPSTNCuda_Measure.cpp

```diff
@@ -92,3 +92,68 @@ TEMPLATE_TEST_CASE("Probabilities", "[Measures]", float, double) {
         REQUIRE_THROWS_AS(measure.probs({2, 1}), LightningException);
     }
 }
+
+TEMPLATE_TEST_CASE("Samples", "[Measures]", float, double) {
+    using TensorNetT = MPSTNCuda<TestType>;
+
+    constexpr uint32_t twos[] = {
+        1U << 0U,  1U << 1U,  1U << 2U,  1U << 3U,  1U << 4U,  1U << 5U,
+        1U << 6U,  1U << 7U,  1U << 8U,  1U << 9U,  1U << 10U, 1U << 11U,
+        1U << 12U, 1U << 13U, 1U << 14U, 1U << 15U, 1U << 16U, 1U << 17U,
+        1U << 18U, 1U << 19U, 1U << 20U, 1U << 21U, 1U << 22U, 1U << 23U,
+        1U << 24U, 1U << 25U, 1U << 26U, 1U << 27U, 1U << 28U, 1U << 29U,
+        1U << 30U, 1U << 31U};
```

Comment:
Could we wrap this in a function (maybe in `pennylane_lightning/core/src/utils/TestHelpers.hpp`) and replace all instances of this array?